aqa-tests/functional/jtreg-buffer/build.xml

```diff
@@ -47,7 +47,6 @@
                                                                                would result to run the suite twice  -->
           <getFileWithRetry file="jtreg-buffer"
                command="git clone --depth 1 -q ${GIT_REPO}rh-openjdk/jtreg-buffer.git -b main jtreg-buffer ;  rm -rvf  jtreg-buffer/aqa-tests/" />
-          <checkGitRepoSha repoDir="jtreg-buffer" />
```

Comment:
But they did not removed it wothout repalcement right? There is repalcement....  getSHAS.sh .. but maybe it is used transaprently? Why are gitactions disabled?